package/MDAnalysis/coordinates/XDR.py

```diff
@@ -197,7 +197,7 @@ def _load_offsets(self):
         try:
             with fasteners.InterProcessLock(lock_name) as filelock:
                 pass
-        except PermissionError:
+        except (PermissionError, OSError):
```

Comment:
A reproducer for the original handled PermissionError is straightforward, like below, I wonder if we could cook something up for the other error? May not be worth much energy though.

```diff
--- a/testsuite/MDAnalysisTests/coordinates/test_xdr.py
+++ b/testsuite/MDAnalysisTests/coordinates/test_xdr.py
@@ -904,3 +904,14 @@ class TestTRRReader_offsets(_GromacsReader_offsets):
         9155712, 10300176
     ])
     _reader = mda.coordinates.TRR.TRRReader
+
+
+def test_offset_readonly_handle(tmpdir):
+    with tmpdir.as_cwd():
+        curr_path = os.getcwd()
+        new_gro = os.path.join(curr_path, os.path.basename(GRO))
+        new_xtc = os.path.join(curr_path, os.path.basename(XTC))
+        shutil.copyfile(GRO, new_gro)
+        shutil.copyfile(XTC, new_xtc)
+        os.chmod(curr_path, 0o555)
+        u = mda.Universe(new_gro, new_xtc)

```